package.json

```diff
@@ -1,50 +1,44 @@
 {
-  "name": "infusion",
-  "description": "Infusion is an application framework for developing flexible stuff with JavaScript.",
-  "version": "2.0.0",
-  "author": "Fluid Project",
-  "bugs": "http://issues.fluidproject.org/browse/FLUID",
-  "homepage": "http://www.fluidproject.org/",
-  "dependencies": {
-      "resolve": "1.1.6"
-  },
-  "licenses": [
-    {
-      "type": "BSD-3-Clause",
-      "url": "http://www.opensource.org/licenses/BSD-3-Clause"
+    "name": "infusion",
+    "description": "Infusion is an application framework for developing flexible stuff with JavaScript",
+    "version": "2.0.0",
+    "author": "Fluid Project",
+    "bugs": "http://issues.fluidproject.org/browse/FLUID",
+    "homepage": "http://www.fluidproject.org/",
+    "dependencies": {
+        "resolve": "1.1.6"
     },
-    {
-      "type": "ECL-2.0",
-      "url": "http://www.opensource.org/licenses/ECL-2.0"
+    "license": "(BSD-3-Clause OR ECL-2.0)",
```

Comment:
Is there a reason to change the ```license``` block to this simplified format. The old structure that has the two types individually with corresponding links to license descriptions seems more informative.